packages/rocketchat-ui/client/lib/textarea-autogrow.js

```diff
@@ -6,22 +7,20 @@
 	 * http://github.com/jaz303/jquery-grab-bag/tree/master/javascripts/jquery.autogrow-textarea.js
 	 */
 	$.fn.autogrow = function(options) {
+		var shadow = $("div.autogrow-shadow");
```

Comment:
Can we use `let` here?

Reply:
Maybe we could add a `TODO` and investigate if there is a way to pass the container element to search this `.autogrow-shadow` class. Doing like this we are looking at the whole DOM.

Reply:
actually this code was a copy/paste thats why I'm not using >ES5 on code (just were I changed)

Reply:
and about the find, we can use `body > .autogrow-shadow` or use an `id` instead, but this code just runs "one time"(one time per opened room) so I dont think will change a lot, but I will change